docs/PacketFence_Rapid7NexposeInsightVM_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,185 @@
+* In the *Menu*, select *Engines* and define the engine, depending on the licence and scope you have.
+
+How to sent Syslogs to PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* On the left, create a new alert.
+* In the *Menu*, select now *Alerts*
+
+|===
+|*Enable*: Must be checked. 
+
+*Alert Name*: Rsyslog to PacketFence or else.
+
+*Maximum Alerts to send*: blank (none)
+
+*Scan events*: Check all.
+
+*Vulnerability Events*: _Any severity_ ; Check as well _Confirmed_, _Unconfirmed_, _Potential_
+
+*Notification Method*: Select _Syslog message_
+
+*Syslog Server*: "PacketFence cluster VIP or server IP for a standalone"  
+|===
+
+In the *Menu*, select now *Schedule*, and define the schedules of your audits.
+
+
+Step 2: PacketFence configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+"What need to be done on the PacketFence Server to understand and take action about endpoints containing threats"
+
+Get PacketFence ready to welcome Nexpose Syslogs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+WARNING: If you are using a PacketFence cluster, you will need to do these steps in all your PacketFence servers.
+
+* Logon to PacketFence Server with a _ssh_ terminal.
+* Create the fifo pipe file that PacketFence will use to get datas from Nexpose
+ 
+ mkfifo /usr/local/pf/var/run/nexpose_pipe
+
+* Create a new file named /etc/rsyslog.d/nexpose-log.conf.
+
+ # rsyslog conf for Rapid7 Nexpose server logs reception
+ if $programname == 'Nexpose' then /usr/local/pf/var/run/nexpose_pipe
+ & ~
+
+* Modify /etc/rsyslog.conf to accept syslogs on 'udp 514'
+** Uncomment the two lines:
+
+ $ModLoad imudp
+ $UDPServerRun 514
+
+* Restart the 'rsyslog' service
+** Run this command:
+  
+  service rsyslog restart
+
+At this point Packetfence must be able to get the Nexpose audits results.
+
+TIP: You can see if the Nexpose server is sending to the right server by minitoring both sides traffic, with a 'tcpdump -i any dst host YOUR_PACKETFENCE_SERVERI' on your Rapid7 Nexpose server and 'tcpdump -i any port 514' on the PacketFence server.
+
+Get PacketFence to be able to configure what action need to be taken
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's give PacketFence the Nexpose parser it needs to detect Nexpose events ::
+* Logon to PacketFence Server with a browser on the PacketFenceGUI 
+** _https://YOUR_PACKETFENCE_SERVER_IP:1443_
+
+* Go to *Configuration -> Integration -> Syslog parsers*
+* On the drop list 'ADD SYSLOG PARSER', take 'Nexpose'
+* As Detector, put the name of your choice for this parser.
+* In Alert pipe, put the 'absolute' path to our nexpose pipe
+
+  /usr/local/pf/var/run/nexpose_pipe
+
+* Click on 'SAVE'.
+
+Once done, we need to restart the good services ::
+* Restart the 'pfdetect' and the 'pfqueue' service.
+----
+/usr/local/pf/pfcmd service pfdetect restart
+/usr/local/pf/pfcmd service pfqueue restart
+----
+
+Now that PacketFence got the datas to process, lets tell it what to do with it ::
```

Comment:
'got the datas to process, lets tell it what to do with it ::' ->

is properly configured to receive information from Nexpose, lets configure it to perform some actions: